tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -251,6 +252,10 @@ install_cc_sev_ovmf(){
  	install_cc_tee_ovmf "sev" "edk2-sev.tar.gz"
 }
 
+install_cc_x86_ovmf(){
+ 	install_cc_tee_ovmf "" "edk2-x86_64.tar.gz"
```

Comment:
Hi @Alex-Carter01 , to use `install_cc_tee_ovmf()` you will need to pass a `tee` value otherwise the function will not get the ovmf url/version correctly (see https://github.com/kata-containers/kata-containers/blob/CCv0/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L469)

Reply:
Has this been resolved? I now specify x86_64 ovmf_build